index.html

```diff
@@ -319,6 +324,31 @@
         </ul>
       </article>
 
+      
+      <article class="import">
+        <label>Import</label>
+        <ul>
+          <li>
+            <div class="container" >
+              <input type="file" name="file" id="file">
+              
+              <!-- Drag and Drop container-->
+              <div class="upload-area"  id="uploadfile">
```

Comment:
Maybe rename this to "70U Import" so people not familiar with the context have an idea? I can add a section to the README just in case someone actually clicks it :)